src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -912,17 +914,17 @@ def perturb_individual_age(self, pop: pd.DataFrame) -> pd.Series:
 
             # Calculate shifted ages and see if any are negative and need to be resampled.
             shifted_ages = pop.loc[to_shift, "age"] + age_shift
-            negative_ages_mask = shifted_ages < 0
-            pop.loc[~negative_ages_mask & to_shift, "age"] = shifted_ages.loc[
-                ~negative_ages_mask & to_shift
-            ]
-            to_shift = negative_ages_mask
+            positive_ages_idx = shifted_ages.loc[shifted_ages >= 0].index
```

Comment:
This is better!

Hah, kinda a nit that you call it `positive_ages_idx` but they include 0 (ie, not strictly positive values)...but i also can't think of a better name.